engine-nextgen/src/main/kotlin/graphql/nadel/validation/NadelTypeValidation.kt

```diff
@@ -167,8 +167,12 @@ internal class NadelTypeValidation(
             .filter { (key) ->
                 key in nameNamesUsed
             }
+            .filterNot { (key) ->
+                key in allNadelBuiltInTypeNames
```

Comment:
The new type resolution pull in a lot more types, let's ignore all the built in ones.